src/xdocs/checks/javadoc/javadocleadingasteriskalign.xml.template

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document xmlns="http://maven.apache.org/XDOC/2.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/XDOC/2.0 https://maven.apache.org/xsd/xdoc-2.0.xsd">
+  <head>
+    <title>JavadocLeadingAsteriskAlign</title>
+  </head>
+  <body>
+    <section name="JavadocLeadingAsteriskAlign">
+      <p>Since Checkstyle 10.17.0</p>
+      <subsection name="Description" id="Description">
+        <p>
+          Checks the alignment of
+          <a href="https://docs.oracle.com/en/java/javase/14/docs/specs/javadoc/doc-comment-spec.html#leading-asterisks">
+          leading asterisks</a> in a Javadoc comment. The Check ensures that leading asterisks
+          are aligned vertically under the second asterisk ( &#42; )
+          of opening Javadoc tag. The alignment of closing Javadoc tag ( &#42;/ ) is also checked.
+          If a line contains a non-whitespace character before the
+          leading asterisk then that line will be ignored. Same applies for closing Javadoc tag.
+          If the ending javadoc line contains a leading asterisk,
+          then that leading asterisk's alignment will be considered,
+          the closing Javadoc tag will be ignored.
+        </p>
+        <p>
+          If you're using tabs then specify the the tab width in the
+          <a href="https://checkstyle.org/config.html#tabWidth">tabWidth</a> property.
+        </p>
+      </subsection>
+
+      <subsection name="Properties" id="Properties">
+        <div class="wrapper">
+          <macro name="properties">
+            <param name="modulePath"
+                   value="src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMissingLeadingAsteriskCheck.java"/>
```

Comment:
You need to look over what you wrote and make sure you don't have any copy/paste errors. This check is not `JavadocMissingLeadingAsteriskCheck`.

Reply:
Fixed. thanks for pointing this out.